client/checkout/blocks/index.js

```diff
@@ -65,8 +65,6 @@ registerPaymentMethod( {
 	},
 } );
 
-registerExpressPaymentMethod( paymentRequestPaymentMethod( api ) );
-
 // Call handlePlatformCheckoutEmailInput if platform checkout is enabled and this is the checkout page.
 if ( getConfig( 'isPlatformCheckoutEnabled' ) ) {
```

Comment:
@asumaran I didn't see a way to explicitly set the ordering of these methods with `registerExpressPaymentMethod()` so I went ahead with changing the order in which we register them. LMK your thoughts on this approach.

Reply:
I see no harm on doing this.